src/core/RadioButton/__tests__/RadioButton.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import RadioButton from '../../RadioButton';
+import { render, cleanup, fireEvent } from '@testing-library/react';
+import { KuberaThemeProvider } from '../../../theme';
+import { screen } from '@testing-library/dom';
+
+afterEach(cleanup);
+jest.useFakeTimers();
+
+describe('Radio Button Component', () => {
+  it('Renders', () => {
```

Comment:
can you add some other test cases to check if the radio buttons work in groups as well @saten16